tests/unit/src/utils/test_rlp.py

```diff
@@ -0,0 +1,106 @@
+import random
+
+import pytest
+import pytest_asyncio
+from rlp import decode, encode
+
+random.seed(0)
+
+
+@pytest_asyncio.fixture
+async def rlp(starknet):
+    return await starknet.deploy(
+        source="./tests/unit/src/utils/test_rlp.cairo",
+        cairo_path=["src"],
+        disable_hint_validation=False,
+    )
+
+
+@pytest.fixture
+def long_list():
+    return list(
+        encode(
+            [
+                "0x",
+                "0x0430e23400",
+                "0x5208",
+                "0xdc544d1aa88ff8bbd2f2aec754b1f1e99e1812fd",
+                "0x01",
+                "0x",
+                "0x1b",
+                "0x110d8fee1de53df0870e6eb599ed3bf68fb3f1e62c82dfe5976c467c97253b15",
+                "0x3450b73d2aef2009f026bcbf097a257ae7a37eb5d3b73dc0760aefad2b98e327",
+            ]
+        )
+    )
+
+
+@pytest.fixture
+def short_list():
+    # rlp lib uses always long lists
+    # used https://toolkit.abdk.consulting/ethereum#rlp
+    # to encode: ["0x","0x0430e23400","0x5208","0x01","0x","0x1b","0x110d8fee1de53df0870e6eb599ed3bf68fb3f1e62c82dfe5976c467c97253b15"]
+    return list(
+        bytes.fromhex(
+            "ee80850430e2340082520801801ba0110d8fee1de53df0870e6eb599ed3bf68fb3f1e62c82dfe5976c467c97253b15"
+        )
+    )
+
+
+# using https://etherscan.io/getRawTx?tx= to get samples
+RAW_TX_SAMPLES = [
```

Comment:
looks like we could use the lib instead of these copy/pasted bytes?
https://pyrlp.readthedocs.io/en/latest/quickstart.html